packages/docs-reanimated/docs/scroll/useAnimatedScrollHandler.mdx

```diff
@@ -4,126 +4,110 @@ sidebar_position: 3
-        sv.value = state;
-      },
-    },
-    dependencies
-  );
-  //...
-  return <></>;
-};
-```
-
-`dependencies` here may be:
-
-- `undefined`(argument skipped) - worklet will be rebuilt if there is any change in any of the callbacks' bodies or any values from their closure(variables from outer scope used in worklet),
-- empty array(`[]`) - worklet will be rebuilt only if any of the callbacks' bodies changes,
-- array of values(`[val1, val2, ..., valN]`) - worklet will be rebuilt if there is any change in any of the callbacks bodies or in any values from the given array.
+Only relevant when using Reanimated [without the Babel plugin on the Web.](/docs/guides/web-support/#web-without-the-babel-plugin)
 
 ### Returns
 
-The hook returns a handler object that can be hooked into a scrollable container.
-Note that in order for the handler to be properly triggered, you should use containers that are wrapped with `Animated` (e.g. `Animated.ScrollView` and not just `ScrollView`).
-The handler should be passed under `onScroll` parameter regardless of whether it is configured to receive only scroll or also momentum or drag events.
-
-### Remarks
-
-- The hook returns a handler that may be passed to multiple components. In such situation, the handler will invoke for the given events each time any of the components dispatches them.
+The hook returns a handler object that can be hooked into a scrollable container. The returned handler should be passed under `onScroll` parameter regardless of whether it is configured to receive only scroll or also momentum or drag events. In order for the returned handler to be properly triggered, you should use containers that are wrapped with `Animated` (e.g. `Animated.ScrollView` and not just `ScrollView`).
 
 ## Example
 
-In the below example we define a scroll handler by passing a single worklet handler.
-The worklet handler is triggered for each of the scroll events dispatched to the `Animated.ScrollView` component to which we attach the handler.
+import AnimatedScrollHandler from '@site/src/examples/AnimatedScrollHandler';
+import AnimatedScrollHandlerSrc from '!!raw-loader!@site/src/examples/AnimatedScrollHandler';
 
-```jsx {10-12,29}
-import Animated, {
-  useSharedValue,
-  useAnimatedStyle,
-  useAnimatedScrollHandler,
-} from 'react-native-reanimated';
+<InteractiveExample
+  src={AnimatedScrollHandlerSrc}
+  component={AnimatedScrollHandler}
+/>
 
-function ScrollExample() {
-  const translationY = useSharedValue(0);
+## Remarks
 
-  const scrollHandler = useAnimatedScrollHandler((event) => {
-    translationY.value = event.contentOffset.y;
-  });
+- The returned handler may be passed to multiple components. In such situation, the handler will invoke for the given events each time any of the components dispatches them.
+- If a single worklet function of type `(event) => void` is passed instead of a map of functions matched to event keys, it's treated as a handler for the 'onScroll' event.
```

Comment:
NIT: highlight to make these two `onScrolls` look consistent:
<img width="323" alt="image" src="https://github.com/user-attachments/assets/162b1e4a-c338-4e92-a88d-ec20fd91c7e8">

```suggestion
- If a single worklet function of type `(event) => void` is passed instead of a map of functions matched to event keys, it's treated as a handler for the `onScroll` event.
```